### DISCUSSION TITLE
Feature Request: Calling up a interface when autofill doesn't detected a field to fill.

### DISCUSSION THREAD

**Discussion by KemonoElite** [zeapo/Android-Password-Store#1841] [Q&A]
The Issue

There are some instances where you can't pull up a way to use the password-store, For instance Zerobooks or Gmail won't pull up the search for autofill at all. I am unsure if there is a actual solution to this issue. But I do know of a way around instances that samsung use's.

They of course of there own ecosystem and their keyboard is in that aswell.

But you can call up "Samsung Pass" via the keyboard hotbar and it will pull up card/passwords/etc. You tap on passwords and you can hit the user/Gmail input and click it via the keyboard and click the password input and then hit the password you want to that.

![Screenshot_20220413-172317](https://user-images.githubusercontent.com/97435466/163282337-63fa6016-c853-439c-a6a8-04fd8167c61d.jpg)

![Screenshot_20220413-191442_Google Play services](https://user-images.githubusercontent.com/97435466/163289843-93c0a745-6684-4803-b694-3762e94d722b.jpg)

Now what's my take on it?

I am unsure if there is any simular feature that I haven't seen as of yet hence why it is not a feature request, if there is no solution to this problem I think having a buble to click on on-top of what you are on and be able to go thru to the thing you need and be able to paste it into the field that was last clicked.

Now please keep in mind that I am no developer and I am just essentially tech-savvy. So how it's implemented is beyond my capacity at this very moment.

Any discussion on this to inform me or on this feature I am discussion would be great!

I appreciate your time to read this.

**Comment by msfjarvis** [comment]
You can long press in a text field and the resulting popup should contain an 'Autofill' option which will trigger Password Store.

**Reply by KemonoElite** [reply]
> You can long press in a text field and the resulting popup should contain an 'Autofill' option which will trigger Password Store.

Actually, I have tried this on both applications and it does not seem to do anything. 

I long press, get the autofill option, click it, and does nothing.

I do have logins for these may I add

**Reply by msfjarvis** [reply]
If Password Store cannot launch itself then this is almost certainly something Samsung is causing. If you're able to capture system logs and email them to googleplay at passwordstore dot app I can try to see what's wrong.

**Reply by KemonoElite** [reply]
> If Password Store cannot launch itself then this is almost certainly something Samsung is causing. If you're able to capture system logs and email them to googleplay at passwordstore dot app I can try to see what's wrong.

Sounds good I'll do that.
Mostly everything else works so I wouldn't have expected that.
